examples/angular-cli/README.md

```diff
@@ -1,27 +1,162 @@
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
+_webpack.config.dev.js_
 
-## Running unit tests
+```js
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        loader: "postcss-loader",
+        options: {
+          ident: "postcss",
+          syntax: "postcss-scss",
+          plugins: () => [
+            require("postcss-import"),
+            require("tailwindcss"),
+            require("autoprefixer")
+          ]
+        }
+      }
+    ]
+  }
+};
+```
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+_webpack.config.prod.js_
 
-## Running end-to-end tests
+```js
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        loader: "postcss-loader",
+        options: {
+          ident: "postcss",
+          syntax: "postcss-scss",
+          plugins: () => [
+            require("postcss-import"),
+            require("tailwindcss"),
+            require("autoprefixer"),
+            require("@fullhuman/postcss-purgecss")({
+              // Specify the paths to all of the template files in your project
+              content: [
+                "./src/**/*.html",
+                "./src/**/*.ts"
+                // etc.
+              ],
+              // Include any special characters you're using in this regular expression
+              defaultExtractor: content =>
+                content.match(/[A-Za-z0-9-_:/]+/g) || []
+            })
+          ]
+        }
+      }
+    ]
+  }
+};
+```
 
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
+Then in **angular.js** file, configure the app to use them:
 
-## Further help
+```diff
+"architect": {
+  "build": {
+-   "builder": "@angular-devkit/build-angular:browser",
++   "builder": "@angular-builders/custom-webpack:browser",
+    "options": {
++     "customWebpackConfig": {
++       "path": "./webpack.config.dev.js"
++     },
+      "outputPath": "dist/angular-cli",
+      "index": "src/index.html",
+      "main": "src/main.ts",
 
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+      (...)
+
+    "configurations": {
+      "production": {
++       "customWebpackConfig": {
++         "path": "./webpack.config.prod.js"
++       },
+        "fileReplacements": [
+          {
+            "replace": "src/environments/environment.ts",
+            "with": "src/environments/environment.prod.ts"
+          }
+```
+
+Make those changes for both **architect.build** and **architect.serve**.
+
+Next, create the Tailwind config file:
+
+```sh
+npx tailwind init
+```
+
+Finally, add this to `src/styles.css`:
```

Comment:
> Finally, add this to src/styles.css:

Should this not be .scss?

Reply:
Depending on your stylesheet format you select for your Angular app `ng new APPNAME` supports 

* CSS
* SCSS
* Sass
* Less
* Stylus

Reply:
Maybe the explanation should include these as a clarification. Also, has anyone tested this method works the save on all of those (CSS, SCSS, Sass, Less, Stylus) by just adding the imports to the right `styles.[extension]` file, without changing any of the other steps?
If changes are required for some of the style versions we should mention it, or with any extra steps.
Or if some style is not supported we should also mention it.

Reply:
@marcjulian I was thinking more inline withe the context of the document, which references `.scss` everywhere else

Reply:
Sorry for delay. @marcjulian is right you can use any of those but this working example is about **scss** as it starts with `ng new angular-cli --style=scss`. So yes @blowsie it is a typo that I fixed in a later commit back in september (https://github.com/tailwindlabs/tailwindcss-setup-examples/pull/35/commits/8c7902afdbafe590eae98790af24114e3c9b5105).